### PR TITLE
fix(cli): add error message for hlx up command in submodule repo

### PR DESCRIPTION
**Pull request by maxakuru**
## Related Issues
#1794

### CODE REVIEW

File: src/up.cmd.js
Comment (tripodsan) [inline src/up.cmd.js:184]:
```suggestion
      const stat = await fse.lstat(path.resolve(this.directory, '.git'));
```

**Review by tripodsan — DISMISSED** [review]
I would use `path.resolve()`. otherwise, LGTM.

**Comment by maxakuru** [comment]
@tripodsan what's the process for merges? squash? let approver merge?

**Comment by tripodsan** [comment]
> @tripodsan what's the process for merges? squash? let approver merge?

no, it's usually the PR author that does the merge. if squash or merge is a personal choice :-) but all commits need to adhere to the commit message rules (https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular)

so if you have a lot for small (`wip`) commits, you should either squash them before the merge, or do a sqash commit in github. but if you have a _nice_ set of commits, feel free to just merge them.

note, that semantic-release will create a released base on the commit types. so if you have a `fix: ...` commit in the history, if will create a release, of all the commits are only `chore:` it will not create a release.

**Review by tripodsan — APPROVED** [review]

**Pull request merged**
